### PR TITLE
CGP 0054: Reopening the CCF

### PR DESCRIPTION
**Pull request by prezenti**
This proposal is regarding the reopening of the CCF that was created in CGP 17. The fund has been closed to new applicants while the initial stewards wind down their term and close it after current grants are completed. This proposal at a high level is for:

- Using the initial fund as a template
- Change of stewards
- Request for additional funding via spend approval of the Celo Community Fund 

The initial discussion regarding this proposal took place on the [Celo forum](https://forum.celo.org/t/reopening-of-the-celo-community-fund/3475)

### CODE REVIEW

**Review by willkraft — APPROVED** [review]

**Comment by aaronmboyd** [comment]
Great work Wade @prezenti

File: CGPs/cgp-0054.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
+---
+cgp: 17
+title: Reopening of the Celo Community Fund
+date-created: 
+author: @MayaRB#2593 @Wade | TPT#4475 @Thylacine | VladiatorLabs.io#6470
```
Comment (aaronmboyd) [inline CGPs/cgp-0054.md:5]: I'm a member of the Celo git organisation under @aaronmboyd so might be worthwhile adding that also

File: CGPs/cgp-0054.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+**Current work:**  Founder and operator of VladiatorLabs.io, a protocol-aligned infrastructure company currently in the process of incorporating in Switzerland and expanding to other proof-of-stake networks, especially in the Cosmos ecosystem. Vladiator Labs is a founding project in the Staking Defense League, an infrastructure special interests group aimed at promoting decentralization in the staking space.
+Additionally has worked briefly as a contractor within cLabs on some minor projects. Involved in the Celo ecosystem since the incentivised testnet and have been supported by the Celo Foundation as an elected validator since genesis (over two different validator groups). 
+Freelance Solidity, blockchain, database and cloud engineer and recently has worked with RaveSpace on metaverse NFT gallery https://musee-dezentral.com.
+
+**Possible disclosures:** Vladiator Labs is a current CCF1 grant recipient for our block map and attestation tool https://vido.vladiatorlabs.io, a Foundation vote recipient, and a Foundation grant recipient for operating an ODIS signer as part of the phone number privacy module.
```
Comment (aaronmboyd) [inline CGPs/cgp-0054.md:165]: I should also note that I was the lead architect and engineer on a Flori-funded project [Humanity Cash](https://humanity.cash) and helped design and develop in a lesser capacity on the governance token design for [Impact Market](https://www.impactmarket.com/), another Flori venture. These ventures are already commercially funded and in production and will not require community grant funding.

**Comment by Maya-R-B** [comment]
awesome - thanks Wade!

**Pull request merged**